syntax = "proto3";

message OrderRequest {
	int32 order_id = 1;
}

message OrderResponse {
	int32 order_id = 1;
	string status = 2;
}

service OrderService {
	rpc GetOrder(OrderRequest) returns (OrderResponse);
}